src/pages/documentation/turn/resources.astro

```diff
@@ -0,0 +1,65 @@
+---
+import type { IStandardPageMetadata } from '@models/products/sgid/types';
+
+import QuickLinks from '@components/page/QuickLinks.astro';
+import Section from '@components/page/Section.astro';
+import Layout from '@layouts/FullWidth.astro';
+
+import ExternalLink from '@components/page/ExternalLink.astro';
+import { stripeSections } from '@utils/page';
+
+const page: IStandardPageMetadata = {
+  title: 'TURN Resources',
+  pillar: 'TURN',
+  section: stripeSections([
+    {},
+    { title: 'GNSS Network' },
+    { title: 'Troubleshooting TURN' },
+    { title: 'Supported Devices' },
+  ]),
+};
+---
+
+<Layout title={page.title} pillar={page.pillar}>
+  <QuickLinks links={page.section} slot="quick-links" />
+  <Section {...page.section[0]}>
+    <p>
+      Here you will find resources related to our GNSS network. This includes information on how the network itself
+      works, tips for troubleshooting issues that might arise, and details on some of the devices that are supported by
+      our network.
+    </p>
+  </Section>
+  <Section {...page.section[1]}>
+    <p><i>What is a GNSS network and how exactly does it work?</i></p>
```

Comment:
**quibble**: `<i>` should be replaced with `<em>` for the same reason as b and strong

Reply:
And done.